ballerina/data-loaders.bal

```diff
@@ -14,9 +14,92 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/jballerina.java;
+import ballerina/file;
+
 # Represents a data loader that can load documents from various sources.
 public type DataLoader isolated object {
     # Loads documents from a source.
     # + return - document or an array of documents, or an `ai:Error` if the loading fails
     public isolated function load() returns Document[]|Document|Error;
 };
+
+# Dataloader that can be used to load supported file types as a `TextDocument` 
+# Currently only support `pdf`, `docx` and `pptx`
```

Comment:
Dont we need to export pdf documents as BinaryDocuments?
Or is this for extract the text content from a pdf and return